tests/test_evaluator.py

```diff
@@ -160,6 +160,52 @@ def test_bootstrap(self):
         self.assertAlmostEqual(results["accuracy"]["confidence_interval"][1], 0.68326, 5)
         self.assertAlmostEqual(results["accuracy"]["standard_error"], 0.24595, 5)
 
+    def test_perf(self):
+        data = Dataset.from_dict({"label": [1, 0, 0], "text": ["great movie", "great movie", "horrible movie"]})
+        model = AutoModelForSequenceClassification.from_pretrained(self.default_model)
+        tokenizer = AutoTokenizer.from_pretrained(self.default_model)
```

Comment:
Besides the actual pipe loading tests I would use the dummy pipeline to make sure the tests run fast. Otherwise we load distilbert n-times which is not great.

Reply:
sounds good, I'll send the fix in a separate PR since it's not directly related to this one (uless you think I should just deal with it here?)